packages/renderer/src/lib/extensions/dev-mode/DevelopmentExtensionTableActionsColumn.spec.ts

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************
```

Comment:
question: Can we simplify the naming ? I can't even see the full name, maybe we can create subfolders ? 

I think we can simplify the filename if they parent name is explicit, exaloke`dev-mode/extension/EmptyPage.svelte` is explicit enough ? what do you think?